src/components/Card.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import noImage from '../images/no-image.png';
+import { formatDate, getTimeAgo } from '../utils/utils';
+
+import styled from 'styled-components';
+
+const CardImageItem = styled.img`
+  width: 34rem;
+  object-fit: cover;
+
+  @media (max-width: 767px) {
+    width: 32.5rem;
+  }
+`;
+
+const CardContainer = styled.div`
+  width: 34rem;
+  height: 33.4rem;
+  box-shadow: 0px 5px 25px 0px rgba(0, 0, 0, 0.08);
+  border-radius: 1.5rem;
+  overflow: hidden;
+
+  @media (max-width: 767px) {
+    width: 32.5rem;
+    height: 32.7rem;
+  }
+
+  &:hover {
+    border: 2px solid #6d6afe;
+    transition: all 0.2s linear;
+
+    ${CardImageItem} {
+      transform: scale(1.3);
+    }
+  }
+`;
+
+const CardImage = styled.div`
+  display: flex;
+  justify-content: center;
+  width: 34rem;
+  height: 20rem;
+  overflow: hidden;
+
+  @media (max-width: 767px) {
+    width: 32.5rem;
+    height: 19.2rem;
+  }
+`;
+
+const CardContents = styled.div`
+  display: flex;
+  flex-direction: column;
+  row-gap: 1rem;
+  padding: 1.5rem 2rem;
+  font-weight: 400;
+`;
+
+const CardDiffTime = styled.div`
+  color: #666;
+  font-size: 1.3rem;
+`;
+
+const CardDescription = styled.div`
+  color: #000;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+  height: 4.9rem;
+`;
+
+const CardCreatedTime = styled.div`
+  color: #333;
+  font-size: 1.4rem;
+`;
+
+const Card = ({ link }) => {
+  const createdAt = link?.createdAt ? link?.createdAt : link?.created_at;
+  const description = link?.description;
+  const imageSource = link?.imageSource ? link?.imageSource : link?.image_source;
+  const title = link?.title;
+  const url = link?.url;
+  const folderId = link?.folder_id;
+
+  return (
+    <CardContainer>
+      <a href={url} target="_blank" rel="noreferrer">
+        <CardImage>
+          <CardImageItem src={imageSource ? imageSource : noImage} alt={title} />
+        </CardImage>
+        <CardContents>
+          <CardDiffTime>{getTimeAgo(createdAt)}</CardDiffTime>
+          <CardDescription>{description}</CardDescription>
+          <CardCreatedTime>{formatDate(createdAt)}</CardCreatedTime>
+        </CardContents>
```

Comment:
@JeongSabibi 

이건 개인적인 선호인데요, 

1. 어차피 콤포넌트가 Card인만큼, 스타일콤포넌트들에게도 Card라는 prefix가 굳이 붙을 필요가 있을까 싶어요. 
styled components 컨벤션 상에서도 `StyledCard` 등과 같이 네이밍을 붙이곤 하는데요. 만약 콤포넌트 이름과 스타일콤포넌트 이름이 겹치는게 아니라면 prefix는 제외시켜도 될 것 같습니다! 

2. 특정 콘텐츠들을 담는 박스 역할을 하는 콤포넌트는 `xxxBox` 라고 하면 어떨까요? `CardImage` 의 경우, 카드에서 보여지는 이미지로 판단될 수 있는 콤포넌트 이름인데, 실제로는 `CardImageItem`이 이미지를 담당하고 있잖아요.  또한 `CardContents` 의 경우, 카드에 들어가는 콘텐츠를 표현하는데, 여러 유형의 콘텐츠를 각 레이아웃에 배치되는 박스로써의 역할을 하구 있구요.


따라서 조금 네이밍을 바꿔본다면 이런식으로 해보면 어떨까 제안드립니다.

```suggestion
        <ImageBox>
          <Image src={imageSource ? imageSource : noImage} alt={title} />
        </ImageBox>
        <ContentBox>
          <DiffTime>{getTimeAgo(createdAt)}</CardDiffTime>
          <Description>{description}</CardDescription>
          <CreatedTime>{formatDate(createdAt)}</CardCreatedTime>
        </ContentBox>
```